### ISSUE TITLE
Fix color contrast in the sections header

### ISSUE THREAD

**Issue by carlosmperezm** [carlosmperezm/My-Website#9]
.greeting-container > h3: Fix any of the following:
Element has insufficient color contrast of 2.4 (foreground color: #57aae1, background-color: #fff8e7, font size: 18.0pt (24px), font-weight: bold). Expected contrast ratio of 3:1

#technologies-section > h3: Fix any of the following:
Element has insufficient color contrast of 2.36 (foreground color: #57aae1, background-color: #eef8ff, font size: 18.0pt (24px), font-weight: bold). Expected contrast ratio of 3:1

#projects-section > h3: Fix any of the following:
Element has insufficient color contrast of 2.4 (foreground color: #57aae1, background-color: #fff8e7, font size: 18.0pt (24px), font-weight: bold). Expected contrast ratio of 3:1

#experience-section > h3:Fix any of the following:
Element has insufficient color contrast of 2.36 (foreground color: #57aae1, background-color: #eef8ff, font size: 18.0pt (24px), font-weight: bold). Expected contrast ratio of 3:1
